app/controllers/application_controller.rb

```diff
@@ -1,9 +1,50 @@
 class ApplicationController < ActionController::Base
+  include Timer
```

Comment:
create a macro like -
auto_session_timeout :5.minutes

Reply:
Handle before_action in SessionHandler